packages/app/src/cli/utilities/developer-platform-client/app-management-client.ts

```diff
@@ -175,7 +175,10 @@ export class AppManagementClient implements DeveloperPlatformClient {
       const tokenResult = await ensureAuthenticatedAppManagementAndBusinessPlatform()
       const {appManagementToken, businessPlatformToken, userId} = tokenResult
 
-      const userInfoResult = await businessPlatformRequestDoc(UserInfo, businessPlatformToken)
+      const userInfoResult = await businessPlatformRequestDoc(UserInfo, businessPlatformToken, undefined, {
+        cacheTTL: 6 * 60 * 60 * 1000,
```

Comment:
It's a bit of a pain to be passing around `cacheTTL` as a number, which you then have to remember the units of. I would suggest taking a bit of time to go deeper and enable the use of `TimeInterval` for `cacheRetrieveOrRepopulate` as we do [here](https://github.com/Shopify/cli/blob/3e42f574844c30ad30f3386f69bb6ee540e23e07/packages/cli-kit/src/private/node/conf-store.ts#L147) for `runAtMinimumInterval`. That would allow you to change this line to 
```ts
        cacheTTL: {hours: 6},
```
which I think will do wonders for clarity.